cabal.project

```diff
@@ -28,13 +28,14 @@ packages:
          ./plugins/hls-qualify-imported-names-plugin
          ./plugins/hls-code-range-plugin
          ./plugins/hls-change-type-signature-plugin
+         ./plugins/hls-stan-plugin
          ./plugins/hls-gadt-plugin
          ./plugins/hls-explicit-fixity-plugin
          ./plugins/hls-explicit-record-fields-plugin
          ./plugins/hls-refactor-plugin
          ./plugins/hls-overloaded-record-dot-plugin
 
-index-state: 2023-10-06T06:12:29Z
+-- index-state: 2023-10-06T06:12:29Z
```

Comment:
I commented the index-state to get the new stan version.

Reply:
You should just update the index-state. No point in commenting it out and the PR can't be merged without it any way.